gateway-js/src/executeQueryPlan.ts

```diff
@@ -25,6 +25,7 @@ import {
   getResponseName
 } from '@apollo/query-planner';
 import { deepMerge } from './utilities/deepMerge';
+import { isNotNullOrUndefined } from '@apollographql/apollo-tools';
```

Comment:
Could we use the version of this that's defined in this `@apollo/gateway`'s `./utilities/array.ts`, instead?  It's not currently `export`'d from its module, but it could be!

https://github.com/apollographql/federation/blob/3aa1b084b70a139c1ee9885e43e537be18480822/gateway-js/src/utilities/array.ts#L1-L5

cc @michael-watson or @martijnwalraven

Reply:
Just pushed a commit with this change